contracts/OrcaVoteManager.sol

```diff
@@ -54,12 +64,62 @@ contract OrcaVoteManager {
         bool indexed yesOrNo
     );
 
+    event CreateSafe(uint256 indexed podId, address safeAddress);
+
     constructor(OrcaPodManager _podManager, OrcaRulebook _rulebook) public {
         deployer = msg.sender;
         rulebook = _rulebook;
         podManager = _podManager;
     }
 
+    // TODO: onlyProtocol
+    // TODO: onlyCallOnceProtection
+    function setupPodVotingAndSafe(
+        uint256 _podId,
+        uint256 _votingPeriod,
+        uint256 _minQuorum,
+        address _gnosisMasterContract
+    ) public {
+        createVotingStrategy(_podId, _votingPeriod, _minQuorum);
+        createSafe(_podId, _gnosisMasterContract);
+    }
+
+    function createSafe(uint256 _podId, address _gnosisMasterContract)
+        internal
+    {
+        bytes memory data = "";
+        address[] memory ownerArray = new address[](1);
+        ownerArray[0] = address(this);
+
+        // encode the setup call that will be called on the new proxy safe
+        // from the proxy factory
+        bytes memory setupData =
+            abi.encodeWithSignature(
+                functionSigSetup,
+                ownerArray,
+                uint256(1),
+                address(0),
+                data,
+                address(0),
+                address(0),
+                uint256(0),
+                address(0)
+            );
+
+        bytes memory createProxyWithSetupData =
+            abi.encodeWithSignature(
+                functionSigCreateProxy,
+                _gnosisMasterContract,
+                setupData
+            );
+        (bool success, bytes memory result) =
+            proxyFactoryAddress.call(createProxyWithSetupData);
```

Comment:
how come we had to call this with an encoded call, could'nt we have imported `proxyFactory.sol` and just called these like regular contract to contract functions ?